Utilities/Certificates/CreateTrustedCertificate.bat

```diff
@@ -0,0 +1,59 @@
+@echo off
```

Comment:
I get this (also in CMD and as admin):

![image](https://user-images.githubusercontent.com/1976647/111546068-5d3bf300-8777-11eb-88f1-af7dbb0ad226.png)

Should the .ssh folder indeed exist?

Reply:
Well, I'm trying to create it if it doesn't. But that failed for you with "The filename, directory name, or volume label syntax is incorrect."

For me it works. I've tested `mkdir` again now, also with using a path on a different drive - I don't get any errors. Could you please try to run `mkdir %userprofile%\.ssh` directly from your PS prompt on E:\ ? And if that fails, maybe `mkdir "%userprofile%\.ssh"`? Other than that, I have no clue how to fix that problem.

In the worst case, if .ssh does not exist I could create those files in %AppData%...

Reply:
Yeah it's probably because of the different drive. However, do we really need to create the .ssh folder under the user folder? Isn't it a temp folder that can be removed after importing the certificate?

Reply:
Turns out, I had an unbalanced " in the MKDIR call!

But I like the idea with the TEMP folder and implemented that.

Also, added error handling so the script won't print success messages anymore when actually a command failed.

And we'll remove the temporary folder after all is done.